src/api/controllers/userController.ts

```diff
@@ -0,0 +1,42 @@
+import { RequestWithUser } from '../../types/requests.js';
+import { NextFunction, Response } from 'express';
+import { UserService } from '../../services/userService.js';
+import { BaseUser } from '../../types/user.js';
+import { BaseController } from './baseController.js';
+import { UserDataMapper } from '../../data-access/mappers/userDataMapper.js';
+import { EntityNotFoundError } from '../../core/errors/entityNotFoundError.js';
+
+
+export class UserController extends BaseController {
+    private userService: UserService;
+
+    constructor(userServiceInjected: UserService) {
+        super();
+        this.userService = userServiceInjected;
```

Comment:
that could be removed (and line #11) just be extending the definition `constructor(private userServiceInjected: UserService)`